codegen/templates/object.gotpl

```diff
@@ -30,6 +30,8 @@ func (ec *executionContext) _{{$object.GQLType}}(ctx context.Context, sel ast.Se
 		ctx = graphql.WithResolverContext(ctx, &graphql.ResolverContext{
 			Object: {{$object.GQLType|quote}},
 		})
+	{{else}}
+		graphql.GetResolverContext(ctx).Result = obj
```

Comment:
This should probably happen using res around field.gotpl:62, here its going to be repeated for every child

Reply:
ah... make sense... 
It was a bit different from what I thought.
so I'll write what I want to do.